superset/assets/javascripts/components/FilterableTable/FilterableTable.jsx

```diff
@@ -0,0 +1,190 @@
+import { List } from 'immutable';
+import React, { PropTypes, PureComponent } from 'react';
+import {
+  Column,
+  Table,
+  SortDirection,
+  SortIndicator,
+} from 'react-virtualized';
+import { getTextWidth } from '../../modules/visUtils';
+
+const propTypes = {
+  orderedColumnKeys: PropTypes.array.isRequired,
+  data: PropTypes.array.isRequired,
+  height: PropTypes.number.isRequired,
+  filterText: PropTypes.string,
+  headerHeight: PropTypes.number,
+  overscanRowCount: PropTypes.number,
+  rowHeight: PropTypes.number,
```

Comment:
@mistercrunch and i talked offline about this, we'll keep the static row height and add a max column width. for columns that require truncating the text, we will show a modal or popover... currently there is a tooltip when hovering on any cell which shows the content, but i'm not sure if this is clear enough to the user.

![screenshot 2017-04-18 13 57 01](https://cloud.githubusercontent.com/assets/130878/25152760/f2e926ca-243e-11e7-9a5b-b8a8d9077085.png)

i'll tackle setting a max column width in a follow up PR.